packages/myst-directives/src/figure.ts

```diff
@@ -56,6 +56,10 @@ export const figureDirective: DirectiveSpec = {
       type: String,
       doc: 'A placeholder image when using a notebook cell as the figure contents. This will be shown in place of the Jupyter output until an execution environment is attached. It will also be used in static outputs, such as a PDF output.',
     },
+    static: {
```

Comment:
I suspect we do not want to call this `static`. @stevejpurves suggested `hard-copy` in #1157 . We could also do something like `placeholder:pdf`, but really, this applies to all "paper" exports: docx, tex, typst, and pdf. `placeholder:paper`? Not sure how confusing `placeholder` vs `placeholder:pdf` might be.